release-team/role-handbooks/release-lead.md

```diff
@@ -50,12 +50,12 @@ Among the specific time commitments you have are:
 
 ## Choosing a Release Team
 
-One of your first, and definitely your most important, duties as Release Lead is to select a Release Team.  Currently there are 10 roles to fill on the Release Team, plus one or more shadows (trainees) of each of those roles. each of which has specific requirements. Aside from any specifics, in general you are looking for candidates who:
+One of your first, and definitely your most important, duties as Release Lead is to select a Release Team in consultation with SIG-Release.  Currently there are 10 roles to fill on the Release Team, plus one or more shadows (trainees) of each of those roles. each of which has specific requirements. Aside from any specifics, in general you are looking for candidates who:
 
 1. Can commit to the amount of time required across the release cycle;
 2. Are enthusiastic about being on the release team;
-3. Are members of the relevant SIGs for their role;
-4. And have some prior experience with contributing to Kubernetes.
+3. Are members of the relevant SIGs for their role, if applicable;
+4. And have some prior experience with contributing to Kubernetes (such as shadowing a role for a prior release).
```

Comment:
I was thinking something like, eg:

4. Lead roles especially need some prior experience with contributing to Kubernetes (such as shadowing a role for the prior release), but shadowing may represent an entry point or growth opportunity.

Reply:
As is the text seems sufficient.  I'm fine with it either way really, just saying out loud what I thought I heard Caleb and you saying, but didn't end up seeing in the .md text.